### ISSUE TITLE
Rebuilding the cache throws a warning message

### ISSUE THREAD

**Issue by xchs** [contao/core#8150]
Encountered this issue in Contao 3.5.6 running on PHP 7.0.0.

To reproduce:
* Open the _"Maintenance"_ back end module
* Purge the internal cache
* Rebuild the cache

Right after rebuilding the (internal) cache I instantly get a couple of these warnings:
```code
Warning: mkdir(): File exists in system/modules/core/library/Contao/Files/Php.php on line 33

#0 [internal function]: __error(2, 'mkdir(): File e...', '/var/www/virtua...', 33, Array)
#1 system/modules/core/library/Contao/Files/Php.php(33): mkdir('/var/www/virtua...')
#2 system/modules/core/library/Contao/Folder.php(87): Contao\Files\Php->mkdir('system')
#3 system/modules/core/library/Contao/File.php(554): Contao\Folder->__construct('system')
#4 system/modules/core/library/Contao/Automator.php(505): Contao\File->close()
#5 system/modules/core/library/Contao/Automator.php(478): Contao\Automator->generateConfigCache()
#6 system/modules/core/controllers/BackendMain.php(91): Contao\Automator->generateInternalCache()
#7 contao/main.php(19): Contao\BackendMain->__construct()
#8 {main}
```

**Comment by mroi** [comment]
I see this too and fixed it in #8181.

**Comment by bytehead** [comment]
Confirmed as well from me.

**Comment by leofeyer** [comment]
Fixed in 215df5b1dab74e2830228126aed22dabefe1e63c.
